frontend/src/views/PreparationView.tsx

```diff
@@ -0,0 +1,80 @@
+
+import * as React from 'react';
+import { Chip, NavigationStructure, NavigationDrawer } from '@equinor/fusion-components';
+import { Barrier, Answer, Evaluation, Question, } from '../api/models';
+import QuestionAndAnswerForm from '../components/QuestionAndAnswer/QuestionAndAnswerForm';
+import { Box } from '@material-ui/core';
+
+interface BarrierQuestionsProps
+{
+    title: string
+    questions: Question[]
+}
+
+const BarrierQuestions = ({title, questions}: BarrierQuestionsProps) => {
```

Comment:
Could we rather call this something like 

```suggestion
const BarrierQuestionsView = ({barrier, questions}: BarrierQuestionsViewProps) => {
```

Or `Form` if that is more appropriate?

Reply:
I also think that this could be moved to its own file.